modules/govuk/manifests/node/s_elasticsearch.pp

```diff
@@ -28,6 +31,24 @@
     instances  => $::fqdn,
   }
 
+  case $govuk_elasticsearch::version {
+    /^0.9/: { $cloud_aws_version = '1.16.0' }
+    /^1.0/: { $cloud_aws_version = '2.0.0' }
+    /^1.1/: { $cloud_aws_version = '2.1.1' }
+    /^1.2/: { $cloud_aws_version = '2.2.0' }
+    /^1.3/: { $cloud_aws_version = '2.3.0' }
+    /^1.4/: { $cloud_aws_version = '2.4.2' }
+    /^1.5/: { $cloud_aws_version = '2.5.1' }
+    /^1.6/: { $cloud_aws_version = '2.6.1' }
+    /^1.7/: { $cloud_aws_version = '2.7.1' }
+    default: { $cloud_aws_version = '2.7.1' }
+  }
```

Comment:
It's in the readme for the AWS plugin: https://github.com/elastic/elasticsearch-cloud-aws
You have to match the plugin version with the version of ES, and since we are currently running multiple versions of ES...